src/main/kotlin/app/revanced/patches/reddit/customclients/syncforreddit/api/clientid/fingerprints/GetAuthorizationStringFingerprint.kt

```diff
@@ -1,4 +1,4 @@
-package app.revanced.patches.reddit.customclients.syncforreddit.api.fingerprints
+package app.revanced.patches.reddit.customclients.syncforreddit.api.clientid.fingerprints
```

Comment:
The package has been moved to `clientid` for "Sync for Reddit". But the package has not been changed for all the other 3rd party Reddit clients so that the package is consistent across all the client's patches. Consider moving the packages for all the other "clientId" API patches to the corresponding package.